### PR TITLE
Fix tests and run on a live docker instance

### PR DESCRIPTION
**Pull request by tlunter**
@nahiluhmot @bfulton

Tests do take quite a bit longer, but now we're actually testing against a real server

### CODE REVIEW

**Comment by bfulton** [comment]
Nice! +1

**Comment by smgt** [comment]
Would be nice to get this merged.

File: lib/docker/image.rb [hunk 1]
```diff
@@ -113,8 +113,12 @@ def create(opts = {}, creds = nil, conn = Docker.connection, &block)
         :response_block => response_block(body, &block)
       )
       json = Docker::Util.fix_json(body)
-      image = json.reverse_each.find { |el| el && el.key?('id') }
-      new(conn, 'id' => image && image.fetch('id'), :headers => headers)
+      completions = json.compact.select { |j| j['status'] && j['status'].include?('complete') }
```
Comment (nahiluhmot) [inline lib/docker/image.rb]: `j['status'].to_s.include?('complete')` might be a little more terse.

**Comment by nahiluhmot** [comment]
@tlunter Looks awesome! Excited to be off of VCR.

**Comment by nahiluhmot** [comment]
:+1:

File: lib/docker/image.rb (hunk 1 above)
Comment (tlunter) [inline lib/docker/image.rb]: I think I prefer this since there is a LOT of messages that get returned and usually only a few matter here.

Comment (nahiluhmot) [inline lib/docker/image.rb]: Sounds good, don't alloc strings you don't need to.

**Pull request merged**